ROADMAP.md

```diff
@@ -29,28 +29,24 @@ new Federation 2 (on the [main branch](https://github.com/apollographql/federati
 
 ## What's Next
 
-### Next Release
-
-* `@tag` export to API schema
-* Compose user-defined directives in subgraphs
-
 ### Under Consideration
 
-* `@defer` support in query planning
-* Subscriptions support in query planning
+* Ability to add fields to one subgraph at a time, allowing those fields to be immediately usable by clients. 
+* Subscriptions support in query planning.
 * Entity interfaces can be spread across multiple subgraphs & interface queries with `@interfaceObject` helper.
 * Harmonizing shared value types across subgraphs to a canonical desired state.
-* Advanced caching, auth, demand control, rate limiting, governance, and more!
+* Advanced caching, auth, demand control, rate limiting, governance, and more.
 * Importing shared types into subgraph schemas, to keep things more DRY.
 * Nested `@provides` support beyond what Fed 2 already supports natively.
```

Comment:
Fwiw, I've seen this one ("Nested `@provides` support beyond ...") listed in roadmaps for a while, and I still have no clue what this is referring to. I'm not aware of particular limitations of nested `@provides` in fed 2.

Reply:
Removed, and we can add later if we find out what's missing :)

Reply:
@pcmanus somehow I missed that [you had already fixed this](https://github.com/apollographql/federation/issues/213#issuecomment-1156140565)
> Closing this issue as this is nested provides are supported as of federation 2.0. There is even a unit test demonstrating it: https://github.com/apollographql/federation/blob/main/query-planner-js/src/__tests__/buildPlan.test.ts#L200.

@korinne The example in this test ☝️ is a great example of query path specific `@provides` on a specific Query root field, which the current https://www.apollographql.com/docs/federation/entities-advanced/#using-provides doesn't really highlight. So think we need add some docs that highlight the use of nested `@provides` and then  think we can close out this issue and then list it as fixed in the Fed 2.0 release lower down in the ROADMAP.md.
- https://github.com/apollographql/federation/issues/416